docs: architecture with the DevWorkspace engine (#2125)
codenvy/che-docs: modules/administration-guide/partials/con_gateway.adoc

[id="gateway_{context}"]
= Gateway

The {prod-short} gateway is a Traefik instance applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource.

> The {prod-short} deployment starts it in the `che-gateway` Deployment.

Reviewer: ```suggestion
The {prod-short} operator manages it as the `che-gateway` Deployment.
```
This is a recurring sentence. It's not wrong to say that Che deployment starts the server side components but that's more correct to say that the Che operator controls every server side components.

[id="gateway_{context}"]
= Gateway

> The {prod-short} gateway is a Traefik instance applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource.

Reviewer 2: role-based access control?

Reviewer: @MichalMaler what do you think about this description of the server-side Gateway?
@themr0c maybe we should mention that this server-side Gateway should not be confused with the workspace side Gateway. And that this is the unique external {prod-short} endpoint: every server-side or workspace-side service is exposed using this single endpoint.

Author: Later, with a pull request focused on the workspaces component.

> The {prod-short} deployment starts it in the `che-gateway` Deployment.

Reviewer 2: Jake addressed a similar sentence above. I agree, it's not 100% clear what starts what, what is "it".

> It need access to:

Reviewer 2: "It" means Gateway? 
"Gateway requires/needs access to" could add clarity.

I believe we're encouraged to avoid "need" as a modal and a more "anthropomorphic verb" (expressing feelings/desires) but I don't think that's a rule.
